tests/asconfig/use-consts/asconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "options": {
+    "use": [
+      "A=1",
+      "B=2",
+      "C=3"
+    ]
+  }
+}
```

Comment:
What is the code style for ending newlines in json files?

Reply:
Existing files like `package.json` and `cli/asc.json` have a line break after the `}`. Not sure how important it is, though.

Reply:
I realize there are more important things. Just trying to keep myself consistent.